python/cli/prelude_cli/views/detect.py

```diff
@@ -225,6 +214,8 @@ def describe_activity(controller, days, view, tests, tags, endpoints, status, do
         filters['statuses'] = status
     if dos:
         filters['dos'] = dos
+    if control:
+        filters['control'] = Control[control.upper()].value
```

Comment:
still practicing my python, but the BE is expecting the query to be the text of the control right? For example "crowdstrike"

Reply:
the enum would be ``.name`` otherwise

Reply:
got it. thank you for clarification.